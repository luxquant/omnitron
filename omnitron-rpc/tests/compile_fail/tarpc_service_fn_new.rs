#[omnitron_rpc::service]
trait World {
    async fn new();
}

fn main() {}
