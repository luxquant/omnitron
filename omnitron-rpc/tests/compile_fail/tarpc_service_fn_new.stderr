error: method name conflicts with generated fn `WorldClient::new`
 --> $DIR/omnitron_rpc_service_fn_new.rs:3:14
  |
3 |     async fn new();
  |              ^^^
